app/scripts/modules/core/src/domain/IExecutionStage.ts

```diff
@@ -24,7 +24,6 @@ export interface IExecutionStage extends IOrchestratedItem, IStage {
   after?: IExecutionStage[];
   before?: IExecutionStage[];
   context: IExecutionContext;
-  id: string;
```

Comment:
This is a surprising change, can you explain?

Reply:
I can not... i bet it was a rebase gone awry, will put back, thank you!